example/events/application-installed.yaml

```
@@ -0,0 +1,16 @@
+name: Application Installed
+description: When did the user installed the mobile app?
```

Small suggestion to grammar

```suggestion
description: When did the user install the mobile app?
```